app-account.js

```diff
@@ -94,12 +94,13 @@ app.post('/change-password', (req, res) => {
   }
 
   let hashed = hashPassword(password);
-
+  let token = uuid.v4();
   // Note that this doesn't have a WHERE. This table only ever has 1
   // row (maybe that will change in the future? if this this will not work)
   accountDb.mutate('UPDATE auth SET password = ?', [hashed]);
+  accountDb.mutate('UPDATE sessions SET token = ?', [token]);
 
-  res.send({ status: 'ok', data: {} });
+  res.send({ status: 'ok', data: { token } });
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏why are you returning the `token` here? Is it actually used in the UI?